Graphics/MusicWheelItem Song NormalPart/get_lamp.lua

```diff
@@ -0,0 +1,86 @@
+local args = {...}
+local graphic = args[1]
+local pn = args[2]
+
+LampColors = {
+    [0]={1,1,1,0},
+    color "#ffffff",
+    color "#34bfff",
+    color "#48ff1d",
+    color "#fff568",
+    color "#bfeaff"
+}
+
+
+local function getLamp(self, song)
```

Comment:
We don't need to pass self.

Also I try to use `UpperCamelCase` for function names and `lowerCamelCase` for variable names (although this is inconsistent through the code base fwiw).